crates/modelardb_common/src/types.rs

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2022 The ModelarDB Contributors
```

Comment:
2023?

Reply:
I think the copyright year should stay at 2022 as the file is only marked as new by Git because it has been renamed and changed a small amount in the same PR.